### ISSUE TITLE
Fix Request for MissingEqualsAndHashCodeWithGetterSetter: spurious violations

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#419]
This results in 4 violations:
```
import lombok.Getter;
import lombok.Setter;

@Getter // bad
@Setter // <-- 2nd violation not needed
@NoArgsConstructor // <-- false positive
@AllArgsConstructor // <-- false positive
public class Foo {}
```
